src/Components/Publishing/Sections/ImageWrapper.tsx

```diff
@@ -13,21 +14,77 @@ interface ImageWrapperProps extends React.HTMLProps<HTMLImageElement> {
   index?: number
 }
 
-export const ImageWrapper: React.SFC<ImageWrapperProps> = props => {
-  const { layout, index, ...blockImageProps }: any = props
-  const fullscreen = (
-    <Fullscreen>
-      <ViewFullscreen index={index} />
-    </Fullscreen>
-  )
-  const viewFullscreen = layout !== "classic" ? fullscreen : false
-
-  return (
-    <StyledImageWrapper>
-      <BlockImage {...blockImageProps} />
-      {viewFullscreen}
-    </StyledImageWrapper>
-  )
+
+
+export class ImageWrapper extends React.Component<Props, any> {
```

Comment:
All images that use `<ImageWrapper />` now fade in. We can update this at some later point

Reply:
Also, i skipped the library route to keep this simple